python/astro_metadata_translator/translators/helpers.py

```diff
@@ -69,6 +69,20 @@ def is_non_science(self: MetadataTranslator) -> None:
     return
 
 
+def no_defocal_info(self: MetadataTranslator) -> None:
```

Comment:
This is not needed if there is an explicit focus_z translator.

Reply:
Removed this after adding default implementation to `MetadataTranslator`.